### ISSUE TITLE
Rounded Corners are ignored on the PolicyKit Authentication Agent window

### ISSUE THREAD

**Issue by jaredmontoya** [matinlotfali/KDE-Rounded-Corners#198]
**Version: KDE 6.0.3
Screenshot:**
![image](https://github.com/matinlotfali/KDE-Rounded-Corners/assets/49511278/de67a09f-06dc-4806-a1cf-1a565be092c0)
**The program in the screenshot:** [Pamac](https://aur.archlinux.org/packages/pamac-aur)

This error is likely reproducible with any app that causes a policy kit prompt to appear.

I suspect that this could happen due to the empty string in the hard-coded [blacklist](https://github.com/matinlotfali/KDE-Rounded-Corners/blob/master/src/ShapeCornersEffect.cpp#L72) but I did not test this with the empty string removed so I might be wrong.

**Comment by matinlotfali** [comment]
Are you sure you are sending the right screenshot? The screenshot is only asking for a root password to install the package.
What is the error that you are being encountered?

**Comment by jaredmontoya** [comment]
Yes it does ask for the root password but it does not matter. Do you see that the PolicyKit prompt that asks for the password has square edges at the bottom? Before some update it had the rounded corners effect applied to it so it had rounded corners like it should. Now it does not.

**Comment by matinlotfali** [comment]
Okay I now understand the issue.
Can you check the configurations and see if "polkit-kde-authentication-agent-1" is listed in the exception list or not?

**Comment by queenbiscuit311** [comment]
![image](https://github.com/matinlotfali/KDE-Rounded-Corners/assets/88851292/ec951a47-3422-4854-8a68-db5c7b49f160)

I've noticed this same thing. The window doesn't seem to be detected in the first place.

**Comment by jaredmontoya** [comment]
> Okay I now understand the issue. Can you check the configurations and see if "polkit-kde-authentication-agent-1" is listed in the exception list or not?

![image](https://github.com/matinlotfali/KDE-Rounded-Corners/assets/49511278/5570652b-41c2-43de-b277-c48c99fce4ab)

**Comment by matinlotfali** [comment]
It is surprising because mine is rounded. Are you on Wayland? or X11?

**Comment by jaredmontoya** [comment]
> It is surprising because mine is rounded. Are you on Wayland? or X11?

I am running Wayland.

**Comment by jaredmontoya** [comment]
I don't know if this will be helpful but on the day that I created this issue I also noticed that on my second computer despite updating to the latest version of the git AUR package(both computers run arch linux) the effect worked differently.
The plasma shell windows like for example the configuration window for the inline clock widget were not rounded and the PolKit prompt was rounded like it should be while on my main pc plasma shell windows were rounded and the PolKit prompt was not.

I cleared any cache related to building this AUR package, then built it again and it started working like on my main computer: plasmashell windows were fixed(rounded) and PolKit prompt broke.

In short I had to do a clean install on my second computer that I use not as often to get the actual latest version of the AUR package installed.

**Comment by matinlotfali** [comment]
@jaredmontoya  Please test the pull request #203

**Comment by jaredmontoya** [comment]
Thank you @matinlotfali , it works now.
